src/service.js

```diff
@@ -1,166 +1,160 @@
-import LambdaRestClient from '@lambdatest/node-rest-client'
-import logger from '@wdio/logger'
+require("source-map-support/register");
```

Comment:
Let's remove the source map commands. The compiled script should automatically add this information.